dts/arm/microchip/mec1501hsz.dtsi

```diff
@@ -163,6 +163,42 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 		};
+		timer0: timer@40000c00 {
```

Comment:
Don't expose timer instance 0 here. As pointed by @scottwcpg recently it has been decided to use this as part of custom in the updated RTOS driver implementation.